test/ThEffectSpec.hs

```diff
@@ -68,25 +68,30 @@ newtype Newtype2 m a where
 makeSem ''Newtype2
 
 -- Data families -------------------------------------------------------------
--- NOTE: Not supported currently, but may be reconsidered in the future.
 
--- data Instance = ADTI | GADTI | NTI
+data Instance = ADTI | GADTI | NTI | MMI
```

Comment:
What do these constructors mean?

Reply:
Simple **ADT**, **GADT**, **n**ewtype and "`m` **m**atching" **I**nstances - I just needed some names to put in there - they don't have any deeper meanings

Reply:
Should I name them simply `I1`, `I2`, etc.?

Reply:
Ahh, no, it's fine. Makes sense now!